### DISCUSSION TITLE
Under Git bash, the first run is correct, and then starting second run there's encoding issue

### DISCUSSION THREAD

**Discussion by jltchiu** [openai/whisper#69] [General]
Under git bash in Window, if I run twice

The first time will always be alright
whisper input.wav --language Chinese --model medium > test1.txt

And if I run the following command right after
whisper input.wav --language Chinese --model medium > test2.txt
UnicodeEncodeError: 'cp950' codec can't encode character '\u76d8' in position 35: illegal multibyte sequence

Technically it should be the same? But in practice I have not been able to run the second line, and starting from the second run it will have the same issue till I closed the bash screen and start a new bash screen. Then I can run once without problem, then the second run problem happens again. 

Any idea how to fix this?

**Comment by FurkanGozukara** [comment]
I fixed error with manually updating all scripts

**Comment by 381426068** [comment]
I meet the same error code when transcribing the audio file. Someone can help?

Output:
#normal output above...
An error occurred: 'gbk' codec can't encode character '\u30fb' in position 34: illegal multibyte sequence
